pkg/utils/security.go

```diff
@@ -17,41 +17,170 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"io/ioutil"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"strings"
 
 	"github.com/pingcap/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
 
+// TLS ...
+type TLS struct {
+	inner  *tls.Config
+	client *http.Client
+	url    string
+}
+
 // ToTLSConfig generates tls's config.
-func ToTLSConfig(SSLCA, SSLCert, SSLKey string) (*tls.Config, error) {
-	var tlsConfig *tls.Config
-	if len(SSLCA) != 0 {
-		var certificates = make([]tls.Certificate, 0)
-		if len(SSLCert) != 0 && len(SSLKey) != 0 {
-			// Load the client certificates from disk
-			certificate, err := tls.LoadX509KeyPair(SSLCert, SSLKey)
-			if err != nil {
-				return nil, errors.Errorf("could not load client key pair: %s", err)
-			}
-			certificates = append(certificates, certificate)
-		}
+func ToTLSConfig(caPath, certPath, keyPath string) (*tls.Config, error) {
+	return ToTLSConfig(caPath, certPath, keyPath)
+}
```

Comment:
????????

Reply:
a mistake, should be `ToTLSConfigWithVerify(caPath, certPath, keyPath, nil)`